### PR TITLE
MPI - Registro de paciente extranjero con DNI

### PR DESCRIPTION
**Pull request by ma7payne**
### Requerimiento
 
https://proyectos.andes.gob.ar/browse/MPI-403

### Funcionalidad desarrollada 

1. Se realizan casos de testing para el registro de pacientes con DNI argentino

### Requiere actualizaciones en la base de datos

- [ ] Si
- [x] No

### CODE REVIEW

**Comment by silviroa** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [9132](http://condorpiedra.andes.gob.ar:85/9132/mochawesome.html)
CYPRESS RUN: [8279](https://dashboard.cypress.io/projects/xr7gft/runs/8279/specs)
TEST START: 2023-06-29T14:13:53.197Z
TOTAL: 407
SUCCESS: 379
FAIL: 26
SKIPPED: 2

File: cypress/plugins/seed-paciente.js [hunk 1]
```diff
@@ -153,16 +153,25 @@ module.exports.createPaciente = async (mongoUri, params) => {
         // si no tiene notaError y no se envió en params se le asigna string vacío
         dto.notaError = (params.notaError) ? params.notaError : (dto.notaError) ? dto.notaError : '';
 
+        // dto.documento = params.documento;
```
Comment (MarianoCampetella) [inline cypress/plugins/seed-paciente.js:156]: Borrar código comentado!

File: cypress/e2e/core/mpi/mpi-registro-extranjero-con-dni.spec.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
+context('MPI-Registro Paciente Extranjero Con DNI Argentino', () => {
+    let token;
+
+    const pacienteMasculino = { fixture: 'mpi/paciente-validado.json' };
+
+    const pacienteExtranjero = {
+        // id: "5e546977fc9e410a8954ad58",
```
Comment (MarianoCampetella) [inline cypress/e2e/core/mpi/mpi-registro-extranjero-con-dni.spec.js:7]: Si el id no se usa borrar el comentario!

**Review by MarianoCampetella — CHANGES_REQUESTED** [review]

File: cypress/e2e/core/mpi/mpi-registro-extranjero-con-dni.spec.js (hunk 2 above)
Comment (ma7payne) [reply cypress/e2e/core/mpi/mpi-registro-extranjero-con-dni.spec.js]: Tambien borrado!

File: cypress/plugins/seed-paciente.js (hunk 1 above)
Comment (ma7payne) [reply cypress/plugins/seed-paciente.js]: Borrado!

**Review by MarianoCampetella — APPROVED** [review]

**Comment by silviroa** [comment]
USUARIO: cramos  
BUILD NUMBER: [9417](http://condorpiedra.andes.gob.ar:85/9417/mochawesome.html)
CYPRESS RUN: [8554](https://dashboard.cypress.io/projects/xr7gft/runs/8554/specs)
TEST START: 2023-08-14T23:16:31.198Z
TOTAL: 407
SUCCESS: 378
FAIL: 27
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: mchiappe  
BUILD NUMBER: [9424](http://condorpiedra.andes.gob.ar:85/9424/mochawesome.html)
CYPRESS RUN: [8561](https://dashboard.cypress.io/projects/xr7gft/runs/8561/specs)
TEST START: 2023-08-15T14:43:59.676Z
TOTAL: 352
SUCCESS: 318
FAIL: 32
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: mchiappe  
BUILD NUMBER: [9425](http://condorpiedra.andes.gob.ar:85/9425/mochawesome.html)
CYPRESS RUN: [8562](https://dashboard.cypress.io/projects/xr7gft/runs/8562/specs)
TEST START: 2023-08-15T14:56:47.224Z
TOTAL: 352
SUCCESS: 329
FAIL: 21
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: mchiappe  
BUILD NUMBER: [9426](http://condorpiedra.andes.gob.ar:85/9426/mochawesome.html)
CYPRESS RUN: [8563](https://dashboard.cypress.io/projects/xr7gft/runs/8563/specs)
TEST START: 2023-08-15T15:25:56.285Z
TOTAL: 407
SUCCESS: 377
FAIL: 28
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: cramos  
BUILD NUMBER: [9430](http://condorpiedra.andes.gob.ar:85/9430/mochawesome.html)
CYPRESS RUN: [8567](https://dashboard.cypress.io/projects/xr7gft/runs/8567/specs)
TEST START: 2023-08-15T19:43:40.137Z
TOTAL: 407
SUCCESS: 377
FAIL: 28
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: mchiappe  
BUILD NUMBER: [9434](http://condorpiedra.andes.gob.ar:85/9434/mochawesome.html)
CYPRESS RUN: [8571](https://dashboard.cypress.io/projects/xr7gft/runs/8571/specs)
TEST START: 2023-08-16T13:07:59.130Z
TOTAL: 407
SUCCESS: 379
FAIL: 26
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: mchiappe  
BUILD NUMBER: [9551](http://condorpiedra.andes.gob.ar:85/9551/mochawesome.html)
CYPRESS RUN: [8677](https://dashboard.cypress.io/projects/xr7gft/runs/8677/specs)
TEST START: 2023-09-05T12:53:23.118Z
TOTAL: 379
SUCCESS: 342
FAIL: 35
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [9560](http://condorpiedra.andes.gob.ar:85/9560/mochawesome.html)
CYPRESS RUN: [8683](https://dashboard.cypress.io/projects/xr7gft/runs/8683/specs)
TEST START: 2023-09-05T16:50:06.276Z
TOTAL: 407
SUCCESS: 378
FAIL: 27
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [9561](http://condorpiedra.andes.gob.ar:85/9561/mochawesome.html)
CYPRESS RUN: [8684](https://dashboard.cypress.io/projects/xr7gft/runs/8684/specs)
TEST START: 2023-09-05T17:09:48.906Z
TOTAL: 407
SUCCESS: 378
FAIL: 27
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [9562](http://condorpiedra.andes.gob.ar:85/9562/mochawesome.html)
CYPRESS RUN: [8685](https://dashboard.cypress.io/projects/xr7gft/runs/8685/specs)
TEST START: 2023-09-05T17:24:11.327Z
TOTAL: 407
SUCCESS: 375
FAIL: 30
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: cramos  
BUILD NUMBER: [9565](http://condorpiedra.andes.gob.ar:85/9565/mochawesome.html)
CYPRESS RUN: [8688](https://dashboard.cypress.io/projects/xr7gft/runs/8688/specs)
TEST START: 2023-09-05T20:55:16.278Z
TOTAL: 353
SUCCESS: 327
FAIL: 24
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [9567](http://condorpiedra.andes.gob.ar:85/9567/mochawesome.html)
CYPRESS RUN: [8690](https://dashboard.cypress.io/projects/xr7gft/runs/8690/specs)
TEST START: 2023-09-06T14:13:31.853Z
TOTAL: 407
SUCCESS: 378
FAIL: 27
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: mchiappe  
BUILD NUMBER: [9826](http://condorpiedra.andes.gob.ar:85/9826/mochawesome.html)
CYPRESS RUN: [8936](https://dashboard.cypress.io/projects/xr7gft/runs/8936/specs)
TEST START: 2023-10-26T15:15:09.965Z
TOTAL: 407
SUCCESS: 377
FAIL: 28
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: mchiappe  
BUILD NUMBER: [9842](http://condorpiedra.andes.gob.ar:85/9842/mochawesome.html)
CYPRESS RUN: [8950](https://dashboard.cypress.io/projects/xr7gft/runs/8950/specs)
TEST START: 2023-10-30T14:22:12.570Z
TOTAL: 407
SUCCESS: 378
FAIL: 27
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: mchiappe  
BUILD NUMBER: [9843](http://condorpiedra.andes.gob.ar:85/9843/mochawesome.html)
CYPRESS RUN: [8951](https://dashboard.cypress.io/projects/xr7gft/runs/8951/specs)
TEST START: 2023-10-30T16:16:48.586Z
TOTAL: 406
SUCCESS: 378
FAIL: 26
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: mchiappe  
BUILD NUMBER: [9845](http://condorpiedra.andes.gob.ar:85/9845/mochawesome.html)
CYPRESS RUN: [8953](https://dashboard.cypress.io/projects/xr7gft/runs/8953/specs)
TEST START: 2023-10-30T17:29:11.252Z
TOTAL: 308
SUCCESS: 274
FAIL: 33
SKIPPED: 1
:warning::warning::warning::warning:
Hay archivos que no corrieron. Vuelva a intentar.

**Comment by silviroa** [comment]
USUARIO: mchiappe  
BUILD NUMBER: [9846](http://condorpiedra.andes.gob.ar:85/9846/mochawesome.html)
CYPRESS RUN: [8954](https://dashboard.cypress.io/projects/xr7gft/runs/8954/specs)
TEST START: 2023-10-30T17:48:31.685Z
TOTAL: 406
SUCCESS: 377
FAIL: 27
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: mchiappe  
BUILD NUMBER: [9991](http://condorpiedra.andes.gob.ar:85/9991/mochawesome.html)
CYPRESS RUN: [9100](https://dashboard.cypress.io/projects/xr7gft/runs/9100/specs)
TEST START: 2023-11-28T19:08:32.936Z
TOTAL: 363
SUCCESS: 328
FAIL: 33
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: mchiappe  
BUILD NUMBER: [9996](http://condorpiedra.andes.gob.ar:85/9996/mochawesome.html)
CYPRESS RUN: [9105](https://dashboard.cypress.io/projects/xr7gft/runs/9105/specs)
TEST START: 2023-11-29T12:07:52.104Z
TOTAL: 319
SUCCESS: 282
FAIL: 35
SKIPPED: 2
:warning::warning::warning::warning:
Hay archivos que no corrieron. Vuelva a intentar.

**Comment by silviroa** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [10060](http://condorpiedra.andes.gob.ar:85/10060/mochawesome.html)
CYPRESS RUN: [9168](https://dashboard.cypress.io/projects/xr7gft/runs/9168/specs)
TEST START: 2023-12-15T14:29:00.676Z
TOTAL: 372
SUCCESS: 361
FAIL: 9
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [10061](http://condorpiedra.andes.gob.ar:85/10061/mochawesome.html)
CYPRESS RUN: [9169](https://dashboard.cypress.io/projects/xr7gft/runs/9169/specs)
TEST START: 2023-12-15T14:49:08.994Z
TOTAL: 406
SUCCESS: 404
FAIL: 0
SKIPPED: 2

**Pull request merged**